src/core/core.js

```diff
@@ -90,7 +90,12 @@ module.exports = function() {
 				var text = [];
 				text.push('<ul class="' + chart.id + '-legend">');
 				for (var i = 0; i < chart.data.datasets.length; i++) {
-					text.push('<li><span style="background-color:' + chart.data.datasets[i].backgroundColor + '"></span>');
+					if (chart.options.legend.fontFamily) {
```

Comment:
Why `legend.fontFamily` when the fallback uses `legend.labels.fontFamily`? (I can't find `legend.fontFamily` in the doc though).